setup.cfg

```diff
@@ -2,6 +2,8 @@
 universal = 1
 
 [flake8]
-ignore = E127,E128,E265,E302,N803,N804,N806,E731,E402,E266,E305
-max-line-length = 100
-exclude = .git,.ropeproject,.tox,build,.direnv,__pycache__
+ignore = E203, E266, E501, W503, E302
+max-line-length = 80
```

Comment:
black I believe uses `88`, though I see you've ignored `E501` so I guess this line does nothing! :D

Reply:
Is it possible to use explicit names in there?

-ignore = E203, E266, E501, W503, E302
+ignore = rule-number-one, another-rule

Reply:
unfortunately not -- pylint has this but flake8 does not

Reply:
> black I believe uses 88, though I see you've ignored E501 so I guess this line does nothing! :D

This is from the [recommended flake8 config from black](https://github.com/ambv/black/blob/master/.flake8). E501 is ignored, but Bugbear's B901 is enabled, which will warn when the max-line-length exceeds 10% of `max-line-length`.

Reply:
Ah, woops. Forgot to add that to dev-requirements.txt. Will fix this up after work today.